packages/passport/sdk/src/Passport.ts

```diff
@@ -228,10 +228,18 @@ export class Passport {
     } catch (err) {
       logger.warn('Failed to logout from confirmation screen', err);
     }
-    await Promise.allSettled([
-      this.authManager.logout(),
-      this.magicAdapter.logout(),
-    ]);
+
+    if (this.config.oidcConfiguration.logoutMode === 'silent') {
+      await Promise.allSettled([
+        this.authManager.logout(),
+        this.magicAdapter.logout(),
```

Comment:
QQ related to this https://imtbl.slack.com/archives/C06SX4AUBNE/p1712280521367519 - Does this also need to be in the order of

1. `magicAdapter.logout()`
2. `authManger.logout()`

Also, do we need to prevent calling `authManager.logout` if the call to `magicAdapter.logout` failed? Currently, one of these could reject and the other could resolve - we do not check the results